src/server/network.ts

```diff
@@ -326,6 +381,16 @@ export class Response extends SdkObject {
     return this._timing;
   }
 
+  async serverIPAddressAndPort(): Promise<IPAddressAndPort|null> {
```

Comment:
I would call it remoteAddr there too. But let's wait for e.g. Joels opinion there.